build.gradle

```diff
@@ -7,7 +7,7 @@ buildscript {
         google()
     }
     dependencies {
-        classpath "com.android.tools.build:gradle:${project.gradleVersion}"
```

Comment:
why this change?

Reply:
Actually they are one and the same, but still I will revert it and see why the build is failing.